kairos-cli/src/crypto/public_key.rs

```diff
@@ -0,0 +1,24 @@
+use crate::crypto::error::CryptoError;
+use casper_types::bytesrepr::{FromBytes, ToBytes};
+
+#[derive(Clone)]
+pub struct CasperPublicKey(pub casper_types::PublicKey);
+
+impl CasperPublicKey {
+    pub fn from_bytes(bytes: &[u8]) -> Result<Self, CryptoError> {
```

Comment:
Don’t make this part of the inherent implementation, just impl the FromBytes trait if you want to do this

Reply:
`FromBytes` is a trait existing in `casper_types`, and my intention was to make Kairos as much independent as possible.